apps/condo/domains/miniapp/schema/B2BAppNewsSharingConfig.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema miniapp.B2BAppNewsSharingConfig 'publishUrl:Text; previewUrl:Text; getRecipientsUrl:Text;'`
+ */
+
+const { Url, Text } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BApp')
+
+
+/**
+ * News Sharing B2BApp
+ * 
+ * News Sharing B2BApp allow b2b users to share their NewsItem to external source like Telegram or Whatsapp from /news page
+ *
+ *                         [ whatsapp-sharing-miniapp ] -> [ whatsapp ]
+ * [ condo /news page ] ->              ...                    ...
+ *                         [ telegram-sharing-miniapp ] -> [ telegram ]
+ *
+ * To create miniapp that can be embedded to /news page developer should provide API and information defined here
+ * 
+ * @type {GQLListSchema}
+ */
+const B2BAppNewsSharingConfig = new GQLListSchema('B2BAppNewsSharingConfig', {
+
+    schemaDoc: 'News Sharing B2BApp allow b2b users to share their NewsItem to external source (like Telegram) from /news page',
+    fields: {
+        name: {
+            schemaDoc: 'Short and simple name of the external source. For example: Telegram',
+            type: Text,
```

Comment:
Avoid using datatypes directly. For KS5/KS6 compatibility use strings instead.

```suggestion
            type: 'Text',
```